tensorboard/plugins/profile/overview_page/overview-page.html

```diff
@@ -218,20 +247,30 @@
        return (show_top_ops_table ? 'Hide' : 'Show') + ' table';
      },
 
-     _shouldShowErrorMessage: function(data) {
-       if (!data) return true;
-       return data[2].p['error_message'] != '';
-     },
-
      /* Updates view according to new data */
      _updateView: function() {
-       if (!this._data || !this._active) return;
-       var generalAnalysisJson = this._data[0];
-       var inputAnalysisJson = this._data[1];
-       var runEnvironmentJson = this._data[2];
-       var recommendationJson = this._data[3];
+       if (!this._data || !this._active) {
+         return;
+       }
+       this._generalAnalysisJson = this._data[0];
+       this._inputAnalysisJson = this._data[1];
+       this._runEnvironmentJson = this._data[2];
+       const recommendationJson = this._data[3];
+       this._recommendationJson = recommendationJson;
+       this._statement = recommendationJson.p['statement'];
+       if (this._statement.includes('HIGHLY')) {
+         this.customStyle['--summary-color'] = 'red';
+       } else if (this._statement.includes('MODERATE')) {
+         this.customStyle['--summary-color'] = 'orange';
```

Comment:
in the `else` case, you should probably clear this value

Reply:
Done.